<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="create_portofio_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Portfolio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyse_portfolio_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse Portfolio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyse_stocks_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse Stocks</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>901</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyse_crypto_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse Crypto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="simulate_bond_returns_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulate Bond Returns</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyse_currencies_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse Currencies</string>
    </property>
   </widget>
   <widget class="QLabel" name="introduction_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>901</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
